src/dnlib.netcore20.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
It's a library and should target netstandard, not netcoreapp

Reply:
netcoreapp2.0 is not meant for applications, it means this library targets explicitly .NET Core 2.0 as a target framework. Changing this to netstandard2.0 manually (IDEs don't provide this option) doesn't compile, cause there are types and methods missing from System.Reflection and System.Runtime.InteropServices in .NET Standard 2.0, which are to be found in .NET 4.x and .NET Core 2.0.
I can dive into this deeper, but no promise if this can be solved:

**dnlib.netcore20
  MethodTableToTypeConverter.cs(89, 27): [CS1061] 'TypeBuilder' does not contain a definition for 'CreateType' and no extension method 'CreateType' accepting a first argument of type 'TypeBuilder' could be found (are you missing a using directive or an assembly reference?)
  MethodTableToTypeConverter.cs(108, 27): [CS1061] 'TypeBuilder' does not contain a definition for 'CreateType' and no extension method 'CreateType' accepting a first argument of type 'TypeBuilder' could be found (are you missing a using directive or an assembly reference?)
  ModuleDefMD.cs(246, 26): [CS0117] 'Marshal' does not contain a definition for 'GetHINSTANCE'**

I would suppose to introduce a NuGet package and to provide different project files for different targets, when installing from NuGet you will then always get the right dll provided for your solution. As of now, pure netstandard2.0 is not possible unfortunately.

Reply:
I don't know how useful a netcoreapp build would be since .NET Core 2.0 can already load .NET Framework assemblies, most likely also dnlib.dll. A netstandard build however is much more useful since it doesn't require .NET Core.

.NET Framework, .NET Core and Mono all support netstandard 2.0, but netstandard 2,0 is missing some APIs used by dnlib.

I'll have a look at this some time when I have more time and interest. It's better to just support netstandard 2.0 since it can be used by all three major .NET runtimes.